tests/tracking/fluent/test_fluent_autolog.py

```diff
@@ -154,7 +147,9 @@ def test_universal_autolog_calls_specific_autologs_correctly(library, mlflow_mod
         )
 
 
-def test_universal_autolog_calls_pyspark_immediately():
+@mock.patch("mlflow.tracking.fluent.is_in_databricks_runtime")
+def test_universal_autolog_calls_pyspark_immediately_in_databricks(is_in_databricks_runtime):
+    is_in_databricks_runtime.return_value = True
```

Comment:
We can use `mock.patch` here.

```suggestion
    with mock.patch(..., return_value=True):
        ...
```